### ISSUE TITLE
How to send jmeter test results(*.jtl) to hygieia dashbard .

### ISSUE THREAD

**Issue by satyakv786** [capitalone/Hygieia#1228]
I have setup hygieia dashboard and installed hygieia jenkins plugin and would like to send jmeter load test results (*.jtl) to hygieia dashboard.can you please let me know the steps .Is this the right way to set it up in Jenkins ? Please let me know.
![jenkinshygieiaplugin](https://cloud.githubusercontent.com/assets/16315161/23615961/2d205ecc-0256-11e7-804e-32b22cfdd523.JPG)

**Comment by krishs123** [comment]
@satishc1 @tabladrum can you please advise how to proceed ? Are jmeter test results supported by jenkins hygieia plugin ?

**Comment by satishc1** [comment]
@krishs123 Not yet. But wip.

**Comment by krishs123** [comment]
ok thanks for update .

**Comment by satishc1** [comment]
@krishs123 Is your jmeter running on the same server as jenkins? If not how do you get the results to jenkins ? What format is your jmeter output file ?

**Comment by krishs123** [comment]
hi satish,
               yes jmeter is running on the same server as jenkins and jmeter output format is xml .
thanks.

**Comment by tabladrum** [comment]
Do you store the xml in jenkins workspace?

**Comment by krishs123** [comment]
yes ,  all the results are stored in jenkins workspace (same local machine).

**Comment by aamol** [comment]
Hi @krishs123 , Where you able to solve this problem. I am also looking at the similar use case where I want to have the performance data available in Hygieia. Currently I see there is only AppDynamics plugin available under performance .
